WordPress/src/main/java/org/wordpress/android/ui/compose/components/menu/dropdown/Item.kt

```diff
@@ -0,0 +1,100 @@
+package org.wordpress.android.ui.compose.components.menu.dropdown
+
+import android.content.res.Configuration
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.material.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.MenuDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.wordpress.android.R
+import org.wordpress.android.ui.compose.theme.AppColor
+import org.wordpress.android.ui.compose.theme.AppTheme
+
+@Composable
+fun Item(
+    element: MenuElementData.Item,
+    onMenuItemClick: (MenuElementData) -> Unit,
+) {
+    val enabledContentColor = itemContentColor()
+    val disabledContentColor = if (isLightTheme()) {
+        AppColor.Gray10
+    } else {
+        AppColor.Gray50
+    }
+    androidx.compose.material3.DropdownMenuItem(
+        modifier = Modifier
+            .background(itemBackgroundColor()),
+        onClick = {
+            onMenuItemClick(element)
+            element.onClick()
+        },
+        colors = MenuDefaults.itemColors(
+            textColor = enabledContentColor,
+            leadingIconColor = enabledContentColor,
+            trailingIconColor = enabledContentColor,
+            disabledTextColor = disabledContentColor,
+            disabledLeadingIconColor = disabledContentColor,
+            disabledTrailingIconColor = disabledContentColor,
+        ),
+        text = {
+            Text(
+                text = element.text,
+                color = enabledContentColor,
```

Comment:
❓ Will the `textColor` coming from `DropdownMenuItem` be applied properly if you set a color directly in this `Text` composable?

Reply:
Good catch, thanks! Removed `color` from `Text`